Make REMOTE_ADDR optional

The `user_logged_in` signal handler fails during tests when calling `client.force_login(...)` because the request has no IP (`REMOTE_ADDR`).

```py
def user_logged_in(sender, request, user, **kwargs):
        try:
            with transaction.atomic(using=DATABASE_ALIAS):
                login_event = audit_logger.login({
                    'login_type': LoginEvent.LOGIN,
                    'username': getattr(user, user.USERNAME_FIELD),
                    'user_id': getattr(user, 'id', None),
>                   'remote_ip': request.META[REMOTE_ADDR_HEADER]
                })
E               KeyError: 'REMOTE_ADDR'

[snip]/easyaudit/signals/auth_signals.py:21: KeyError
```

Would it be ok to change this to `request.META.get(REMOTE_ADDR_HEADER)`? `remote_ip` is already nullable so from the `LoginEvent` model perspective it is already supported. And the request signal handler already supports this: 

https://github.com/soynatan/django-easy-audit/blob/bbdf834d671811043e7bab4aaeb39903ac21f982/easyaudit/signals/request_signals.py#L44

--- a/easyaudit/signals/auth_signals.py
+++ b/easyaudit/signals/auth_signals.py
@@ -18,7 +18,7 @@ def user_logged_in(sender, request, user, **kwargs):
                 'login_type': LoginEvent.LOGIN,
                 'username': getattr(user, user.USERNAME_FIELD),
                 'user_id': getattr(user, 'id', None),
-                'remote_ip': request.META[REMOTE_ADDR_HEADER]
+                'remote_ip': request.META.get(REMOTE_ADDR_HEADER, "")
             })
     except Exception:
         if should_propagate_exceptions():

